pkg/defaults/defaults.go

```diff
@@ -405,4 +405,7 @@ const (
 
 	// KubeProxyReplacementHealthzBindAddr is the default kubeproxyReplacement healthz server bind addr
 	KubeProxyReplacementHealthzBindAddr = ""
+
+	// EnableBPFBypassFIBLookup instructs Cilium to enable the FIB lookup bypass optimization for nodeport reverse NAT handling.
+	EnableBPFBypassFIBLookup = true
```

Comment:
should be false by default, no?

Could you also update the Documentation/gettingstarted/kubeproxy-free.rst doc so that this knob's purpose is documented for users? Thx

Reply:
If the default is "true" the behavior before these changes will be retained. IE. the optimization will be done by default. That way only the users with a FHRP gateway setup will need to set this to false.

I will take a look at the doc and get that updated!

Reply:
@borkmann would you prefer to have any doc related commits in the same pull, and sqashed into the same single commit?

Reply:
Ack, true, should have had my coffee first. :) Yeah, can be same commit or separate one, but within the same PR would be good.